eras/byron/crypto/cardano-crypto-wrapper.cabal

```diff
@@ -73,7 +73,7 @@ library
         -Wno-safe -Wno-unsafe -Wunused-packages
 
     build-depends:
-        base >=4.14 && <4.19,
+        base >=4.14 && <5,
```

Comment:
Change like this requires a patch version update. Could you please bump up the version of `byron-crypto-wrapper` in the cabal file from `1.5.1.0` to `1.5.1.1`